src/Controls/tests/DeviceTests/Memory/MemoryTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Handlers;
+using Microsoft.Maui.Hosting;
+using Xunit;
+
+namespace Microsoft.Maui.DeviceTests.Memory;
+
+public class MemoryTests : ControlsHandlerTestBase
+{
+	void SetupBuilder()
+	{
+		EnsureHandlerCreated(builder =>
+		{
+			builder.ConfigureMauiHandlers(handlers =>
+			{
+				handlers.AddHandler<Border, BorderHandler>();
+				handlers.AddHandler<CheckBox, CheckBoxHandler>();
+				handlers.AddHandler<Entry, EntryHandler>();
+				handlers.AddHandler<Editor, EditorHandler>();
+				handlers.AddHandler<Label, LabelHandler>();
+				handlers.AddHandler<IContentView, ContentViewHandler>();
+				handlers.AddHandler<Image, ImageHandler>();
+				handlers.AddHandler<RefreshView, RefreshViewHandler>();
+				handlers.AddHandler<IScrollView, ScrollViewHandler>();
+			});
+		});
+	}
+
+	[Theory("Handler Does Not Leak")]
+	[InlineData(typeof(Border))]
+	[InlineData(typeof(ContentView))]
+	[InlineData(typeof(CheckBox))]
+	[InlineData(typeof(Entry))]
+	[InlineData(typeof(Editor))]
+	[InlineData(typeof(Image))]
+	[InlineData(typeof(Label))]
+	[InlineData(typeof(RefreshView))]
+	[InlineData(typeof(ScrollView))]
+	public async Task HandlerDoesNotLeak(Type type)
```

Comment:
I guess this is what this new parameterized test looks like:

![image](https://github.com/dotnet/maui/assets/840039/0c887265-f07f-4d5e-a468-c5c6ab8fb4e1)